sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/GpuBroadcastHashJoinExec.scala

```diff
@@ -128,28 +128,19 @@ case class GpuBroadcastHashJoinExec(
 
     val boundCondition = condition.map(GpuBindReferences.bindReference(_, output))
 
+    lazy val builtTable = {
+      // TODO clean up intermediate results...
```

Comment:
Is there a followup for this or was this planned to be addressed in this PR?

Reply:
That was there before.  It will be fixed when we are able to put broadcast data into the cache that is used for shuffle.